File: Trello.test.php
<?php

require('Trello.class.php');

class TrelloTest extends Trello {


}

$strOrg = 'test';
$strNew = '';
$hash = $hashValue = Trello::hash($strOrg);
Trello::deHash($hash, $strNew);

echo "\nString to hash: " . $strOrg;
echo "\nHash value: " . $hashValue;
echo "\nDeHashed value: " . $strNew;
echo "\n";

// ------

$str = '';
$hash = 25180466553932;
Trello::deHash($hash, $str);
echo "\nThe secret answer is: " . $str;
echo "\n\n";

File: Trello.test.py
from Trello import Trello

class TrelloTest(Trello):
	def __init__(self):
		super(TrelloTest, self).__init__()


strOrg = 'test';
strNew = '';
hash = hashValue = TrelloTest.hash(strOrg);
strNew = Trello.deHash(hash, strNew);

print  "\nString to hash: ", strOrg;
print  "Hash value: ", hashValue;
print  "DeHashed value: ", strNew;

print "-----------------------------"


str = ''
hash = 25180466553932
str = Trello.deHash(hash, str);
print "\nThe secret answer is: ", str;
